### DISCUSSION TITLE
[Question] how to use EmbedsMany with existing record?

### DISCUSSION THREAD

**Discussion by soulbalz** [jenssegers/laravel-mongodb#2033] [General]
Hi today i have a question I have try to use EmbedsMany i have done on the follow documentation but on the model Book can't get any records of reverse data, i also try to create a record on Book model and save with instance but not thing append to User embed record is that bug or i missing something?
<img width="982" alt="Screen Shot 2020-05-14 at 10 28 18" src="https://user-images.githubusercontent.com/10706704/81889442-b07c6c80-95cd-11ea-8820-ca3ebe953392.png">

**Comment by mean-cj** [comment]
i same.
https://github.com/jenssegers/laravel-mongodb/issues/2035
